docs/includes/use-legacy-edge-or-ie.md

```diff
@@ -2,6 +2,9 @@ If your project is node.js-based (that is, not developed with Visual Studio and
 
 1. If your project was *not* created with the [Yeoman generator for Office Add-ins](../develop/yeoman-generator-overview.md) tool, you need to install the office-addin-dev-settings tool. Run the following command in a command prompt.
 
+   > [!NOTE]
```

Comment:
I think this note should be placed before Step 1 (on line 2 in this file), because a customer shouldn't start any of the steps until they understand what the Beta channel is and how to access it.